components/BCRegistriesAccount.vue

```diff
@@ -43,9 +43,9 @@
               target="_blank"
               rel="noopener noreferrer"
             >
-              Go to BC Business Registry
+              <span>Go to BC Business Registry</span>
```

Comment:
Because this text does not immediately follow the anchor's `>`, a space character gets inserted. Putting the text in a span solves this.

And, as previously mentioned, when unit testing, one can use `toBe()` to verify the exact text without having to worry about leading/trailing line breaks or spaces characters, or reverting to using `toContain()` (which is obviously less specific).